kogito-infinispan-persistence-springboot/README.md

```diff
@@ -121,3 +162,7 @@ Last but not least you can complete review user task by
 ```
 curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"review" : "very good work"}' http://localhost:8080/dealreviews/xxx-aaa-cccc/review/xxx-yyy-zzz
 ```
+
```

Comment:
(line 145) 
- we should use `{uuid}` placeholder instead of `xxx-aaa-cccc`
- `curl -H 'Content-Type:application/json' -H 'Accept:application/json' http://localhost:8080/dealreviews/{uuid}/tasks?user=john`

Reply:
Hi @radtriste Done